pulumi/aws/managed-redis-managed-kafka-temporal-clickhouse/src/mds/resources/mds.ts

```diff
@@ -56,6 +55,13 @@ export async function installMds(args: MdsConfig, releaseOpts: pulumi.CustomReso
             pullPolicy: "Always",
           },
         },
+        computeClassConfig: {
+          data: {
+            "moose-compute-class-resources.json": JSON.stringify(
```

Comment:
does byoc have a copy of this resource right so our changes in hosting k8s doens't affect this?